mw-webapp/src/component/editableText/EditableText.tsx

```diff
@@ -72,16 +73,29 @@ export const EditableText = <T extends string | number>(props: EditableTextProps
    * Check type of coming value and convert it to Number if need to use input with type "number"
    */
   const setValue = (value: string) => {
-    const number = Number(value) ?? 0;
-    const updatedValue = props.type === "number" ? number : value;
-    setText(updatedValue as T);
+    const checkedValue = value || 0;
+    setText(checkedValue as T);
+  };
+
+  /**
+   * Formatting value entered into the input
+   */
+  const formatterValue = (value: string|number) => {
+    const formattedValue = FormatterInputValue.defaultFormatter(value);
+    const updatedValue = props.type === "number" ? formattedValue : value;
+    useEffect(() => {
```

Comment:
formatterValue is not a hook. You can't execute useEffect in the plain function

Reply:
A simple function will not solve this problem
"EditableText.tsx:84 Warning: Cannot update a component (`EditableText`) while rendering a different component (`Input`). To locate the bad setState() call inside `Input`, follow the stack trace as described in"